src/state/store.ts

```diff
@@ -106,6 +106,10 @@ const stateSanitizer = (state: any) => {
 export const createStore = () =>
   configureStore({
     reducer: persistedReducer,
+    enhancers: existingEnhancers => {
+      // Add the autobatch enhancer to the store setup
+      return existingEnhancers.concat(autoBatchEnhancer())
```

Comment:
this has some risk associated with it

Reply:
Should only be a perf improvement and recommended in the CHANGELOG:

> RTK Query also can take advantage of the new "auto-batch enhancer" (described below) for some additional perf optimization, and we recommend adding that to your Redux store configuration.

> RTK Query's internals have been updated to mark several key actions as batchable. While the enhancer is purely opt-in, benchmarks indicate that it can help speed up UI performance with RTK Query, especially when rendering many components with query hooks. We recommend adding this enhancer to your store setup if you're using RTK Query:

I'd say lessgetitin while at it